build.install4j

```diff
@@ -334,6 +334,177 @@
+                              <string>HKEY_CLASSES_ROOT</string>
+                            </object>
+                          </void>
+                          <void property="value">
+                            <string>"${installer:sys.installationDir}\TripleA.exe" "%1"</string>
+                          </void>
+                        </object>
+                      </java>
+                    </serializedBean>
+                    <condition />
+                  </action>
+                  <action name="" id="220" customizedId="" beanClass="com.install4j.runtime.beans.actions.control.RunScriptAction" enabled="true" commentSet="false" comment="" actionElevationType="inherit" rollbackBarrier="false" multiExec="false" failureStrategy="1" errorMessage="Failed to create protocol Handler">
+                    <serializedBean>
+                      <java class="java.beans.XMLDecoder">
+                        <object class="com.install4j.runtime.beans.actions.control.RunScriptAction">
+                          <void property="script">
+                            <object class="com.install4j.api.beans.ScriptProperty">
+                              <void property="value">
+                                <string>if(Util.isLinux()){
+    String home = System.getProperty("user.home");
+    File shareFolder = new File(home + "/.local/share");
+    if(System.getenv("XDG_DATA_HOME") != null || (shareFolder.exists() &amp;&amp; shareFolder.isDirectory())){
+        File applicationsDir = new File(shareFolder, "applications");
+        if(!applicationsDir.exists() || !applicationsDir.isDirectory()){
+            applicationsDir.mkdirs();
+        }
+        File desktopFile = Arrays.stream(applicationsDir.listFiles(new FilenameFilter(){
+            @Override
+            public boolean accept(File dir, String name){
+                return name.matches("TripleA[\\-0-9\\.]*\\.desktop");
+            }
+        })).findAny().get();
+        File mimeappsFile = new File(applicationsDir, "mimeapps.list");
+        if(!desktopFile.exists()){
+            return false;
+        }
+        if(!mimeappsFile.exists()){
+            mimeappsFile.createNewFile();
+        }
+        String mimeType = "x-scheme-handler/triplea";
+        StringBuilder builder = new StringBuilder();
+        try(Scanner scanner = new Scanner(desktopFile)){
+            String mimeTypePrefix = "MimeType=";
+            while(scanner.hasNextLine()){
+                String nextLine = scanner.nextLine();
+                if(nextLine.startsWith("Exec=") &amp;&amp; !nextLine.contains("%u")){
+                    builder.append(nextLine).append(" \"%u\"\n");
+                } else if(!nextLine.startsWith(mimeTypePrefix)){
+                    builder.append(nextLine).append('\n');
+                }
+            }
+            builder.append(mimeTypePrefix).append(mimeType);
+        }
+        try(PrintWriter writer = new PrintWriter(desktopFile)){
```

Comment:
Derp.  Of course it does.  Otherwise, it wouldn't be much of an install tool, right? :smiley: